source/main.cpp

```diff
@@ -517,6 +517,18 @@ int main(int argc, char *argv[])
     }
 #endif
 
+#if defined(EXCLUDE_SECURE_ELEMENT)
```

Comment:
Something I also find helpful is the INFO log message on startup that reports on the enabled and disabled features. If you agree, then please add something similar for Secure Element.

```
...
2022-05-20T21:24:42.670Z [INFO]  {Main.cpp}: Jobs is disabled
2022-05-20T21:24:42.670Z [INFO]  {Main.cpp}: Secure Tunneling is enabled
2022-05-20T21:24:42.672Z [INFO]  {Main.cpp}: Device Defender is disabled
2022-05-20T21:24:42.672Z [INFO]  {Main.cpp}: Sample shadow is disabled
2022-05-20T21:24:42.672Z [INFO]  {Main.cpp}: Pub Sub is disabled
2022-05-20T21:24:42.672Z [INFO]  {Main.cpp}: Sensor Publish is disabled
...
```